README.md

```diff
@@ -1,25 +1,91 @@
-# psibase
+<div align="center">
+  <img src="https://about.psibase.io/assets/logo-psibase-green.0714b8ee.svg" />
```

Comment:
Is this link stable?  I'm somewhat concerned that it might be broken on a website update. I'd also prefer to have the image in the repo, to keep the readme more self-contained.

Reply:
Added the image to the repo